src/utils/enum/SUToDoEnum.js

```diff
@@ -4,10 +4,10 @@ const toDoEnum = {
   NOT_STARTED: { order: 1, value: `${D.toDoPrepare}` },
   CONTACT: { order: 2, value: `${D.toDoContact}` },
   SURVEY: { order: 3, value: `${D.toDoSurvey}` },
-  FINALIZE: { order: 4, value: `${D.toDoFinalize}` },
-  TRANSMIT: { order: 5, value: `${D.toDoTransmit}` },
-  SYNCHRONIZE: { order: 6, value: `${D.toDoSynchronize}` },
-  TERMINATED: { order: 7, value: `${D.toDoTerminated}` },
+  TRANSMIT: { order: 4, value: `${D.toDoTransmit}` },
+  SYNCHRONIZE: { order: 5, value: `${D.toDoSynchronize}` },
+  TERMINATED: { order: 6, value: `${D.toDoTerminated}` },
+  FINALIZE: { order: 10, value: `${D.toDoFinalize}` },
```

Comment:
the Finalize instruction concerns the finalization of the questionnaire; the unit's workflow is broader.